variables.tf

```diff
@@ -171,4 +171,16 @@ variable "subnets" {
 variable "tenancy_id" {
   description = "Tenancy OCID"
   type        = string
+}
+
+variable "log_retention_duration" {
+  type        = number
+  default     = 30
+  description = "Log retention duration"
+}
+
+variable "enable_vcn_logging" {
```

Comment:
bring the control variable of the feature before the variable used to configure the feature i.e. define enable_ first and then the retention.